recipe/meta.yaml

```diff
@@ -7,49 +7,68 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ pypi_name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
   sha256: e8e1017b2cf1dda767aef19d2fd9ee5ebe07e050d430f77a0a7c66ba08b8cdae
+  patches:
+    - patches/0001-unvendoring-libsndfile.patch
+    - patches/0002-platform-tag-fix.patch
 
 build:
-  number: 1
-  noarch: python
-  script:
-    # We are currently building a noarch package, but pysoundfile contains a
-    # install-time check for the OS and injects the sf_wchar_open function
-    # conditionally. If this variable is not set, opening files on Windows doesn't work
-    # Fix coming hopefully with https://github.com/bastibe/python-soundfile/pull/288
-    # Original issue on conda-forge: https://github.com/conda-forge/libsndfile-feedstock/issues/14
-    - export PYSOUNDFILE_PLATFORM=win32
-    - python -m pip install . --no-deps -vv
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  # skip win because libsndfile not available
+  skip: true  # [py<36 or s390x or win]
 
 requirements:
+  build:
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
     - pip
-    - python >=3.6
+    - python
     - setuptools
+    - wheel
     - cffi
   run:
-    - python >=3.6
+    - python
     - cffi
+    # numpy is optional, but from the readme it seems
```

Comment:
It is arguably optional. I say arguably because it's kind of needed by default. `SoundFile` methods like `read` and `write` take an `out` parameter that can be used to specify a different type, like `pandans.DataFrame` for example instead of the default `numpy.array`. But in this case, I think it's fine to have `numpy` in the runtime requirements.